### PR TITLE
Add Noplace Support

### PR DESCRIPTION
**Pull request by Brenden2008**
This PR adds Noplace support into WhatsMyName

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -3809,6 +3809,15 @@
         "known" : ["notabug", "hp", "zPlus"],
         "cat" : "coding"
        },
+       {
+        "name" : "Noplace",
+        "uri_check" : "https://api.nospace.app/api/v1/profiles/username/availability?username={account}",
```
Comment (WebBreacher) [inline wmn-data.json:3814]: is there any place where we can look on the web (without the noplace app) and find noplace user profiles?

**Comment by Brenden2008** [comment]
No, Noplace doesn’t have a web app

**Comment by Brenden2008** [comment]
Okay sounds good.

File: wmn-data.json (hunk 1 above)
Comment (Brenden2008) [reply wmn-data.json:3814]: Noplace doesn't have a web app, so unfortunatly there is nowhere you can find noplace profiles without the noplace app

File: wmn-data.json [hunk 2]
```diff
@@ -3809,6 +3809,15 @@
         "known" : ["notabug", "hp", "zPlus"],
         "cat" : "coding"
        },
+       {
+        "name" : "Noplace",
+        "uri_check" : "https://api.nospace.app/api/v1/profiles/username/availability?username={account}",
+        "e_code" : 409,
+        "e_string" : "Username is already taken",
+        "m_code" : 204,
```
[the remark this answers is not in the document]
Comment (Brenden2008) [reply wmn-data.json:3817]: I just pushed changes fixing this. Thank you for letting me know.

**Comment by WebBreacher** [comment]
Added. Thanks for the submission!

**Pull request merged**